research-software-directory/RSD-as-a-service
# UI cookies

Fixes #533 #517 
Update: change of style
<img width="470" alt="image" src="https://user-images.githubusercontent.com/4195550/194845402-9345df14-ea8b-4d2b-b9fe-d6feda750167.png">


<img width="458" alt="image" src="https://user-images.githubusercontent.com/4195550/194386379-0c56a46d-9c57-4b6e-acd3-423271b34396.png">



Changes proposed in this pull request:

* Change the visuals of the cookies banner to make it less intrusive
* Allows interaction on the web (needed for embed pages) and to not user interaction
* It avoids the cookie's consent to appear again when using cookies blockers

How to test:

* delete all cookies and local storage and check if the message still appears after granting consent

PR Checklist:
